R/wtr-layer-segments.R

```diff
@@ -0,0 +1,134 @@
+#' @export
+#' @title Exploration of lake water column layers
+#' @description  Extract water column parameters of a given parameter from a profile using the split-and-merge algorithm. The cline is defined as the midpoint of the layer of water where the physical property change in the greatest over a small difference. The exact cline depends on the specification of measure. For example if temperature is specified,  then we can expect cline to output the thermocline.
```

Comment:
Minor point, but could you add newlines so these don't extend past ~100 characters wide? While a lot of editors can automatically wrap the text, github does not.

Reply:
Addressed here: https://github.com/GLEON/rLakeAnalyzer/pull/85/commits/61a3621ce325eeb4e662a363c30ec0e123a26206